akka-actor/src/main/scala/akka/serialization/Serializer.scala

```diff
@@ -351,6 +351,7 @@ final case class DisabledJavaSerializer(system: ExtendedActorSystem) extends Ser
   }
 
   override def fromBinary(buf: ByteBuffer, manifest: String): AnyRef = {
+    log.warning(LogMarker.Security, "Incoming message is using DisabledJavaSerializer resulting in Message class None! Manifest was: [{}]", manifest)
     this.fromBinary(empty, None)
```

Comment:
@patriknw makes sense, I will make the changes. The original concern I believe was that `... Message class was: [None]` in the logs was somewhat confusing because it was difficult to determine where it was coming from.